.circleci/config.yml

```diff
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
```

Comment:
This step also seems like a candidate for extraction, if easily doable (doubtful tho) 🤔